app/_locales/en/messages.json

```diff
@@ -1729,7 +1729,7 @@
     "message": "MetaMask fee"
   },
   "swapMetaMaskFeeDescription": {
-    "message": "A service fee of $1% is automatically factored into each quote, which supports ongoing development to make MetaMask even better.",
+    "message": "We find the best price from the top liquidity sources, every time. A fee of $1% is automatically factored into each quote, which supports ongoing development to make MetaMask even better.",
```

Comment:
Is the percent sign being duplicated here? I noticed that it's in both of these messages _and_ in the `metaMaskFee` variable passed in.